pkg/controller/certificates/internal/secretsmanager/secret_test.go

```diff
@@ -400,6 +414,47 @@ func TestSecretsManager(t *testing.T) {
 			},
 			expectedErr: false,
 		},
+
```

Comment:
Please add a test for:

"if secret exists with 'tls-combined.pem' and 'tls.der' set, but Certificate has no output formats set, update Secret removing those fields".

We should also have the combination of the above with 2 more tests for removing _just_ `tls-combined.pem'  or just `tls.der`.

Reply:
OK, I've added more tests. Now we have
- if secret does not exists and we want additional output format DER only
- if secret does not exists and we want additional output format CombinedPEM only
- if secret does not exists and we want additional output format DER and CombinedPEM
- if secret exists with tls-combined.pem and der.key and we don't want any additional output format
- if secret exists with tls-combined.pem and der.key and we only want DER additional output format
- if secret exists with tls-combined.pem and der.key and we only want CombinedPEM additional output format